### PR TITLE
Update managing-multiple-accounts.md

### PR DESCRIPTION
**Pull request by joaoreis13**
Changed instructions on how to user GIT helper in MacOS to delete cached authentication.

### Why: Git helpers runs on its specific context and does not receive input direct from command line, therefore you must give to it the input interactively.

Closes: 

### Check off the following:
https://docs.github.com/en/get-started/getting-started-with-git/updating-credentials-from-the-macos-keychain

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@joaoreis13 Thanks so much for opening a PR! I'll get this triaged for review ✨

File: content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts.md [hunk 1]
```diff
@@ -52,10 +52,14 @@ Alternatively, if you want to use the HTTPS protocol for both accounts, you can
 {% data reusables.git.confirm-credential-manager %}
 {% data reusables.git.clear-the-stored-credentials %}
    {% data reusables.git.no-credential-manager %}
-   * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, enter the following command.
+   * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, enter the following command and interactive inputs:
```
Comment (subatoi) [inline content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts.md:55]:
```suggestion
   * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, you can use the credential helper on the command line:
```

File: content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts.md [hunk 2]
```diff
@@ -52,10 +52,14 @@ Alternatively, if you want to use the HTTPS protocol for both accounts, you can
...
 
      ```shell copy
-     git credential-osxkeychain erase https://github.com
+     > git credential-osxkeychain erase
+     host=github.com
+     protocol=https
+     ENTER
```
Comment (subatoi) [inline content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts.md:61]:
```suggestion
     ```suggestion
     $ git credential-osxkeychain erase
     host={% data variables.product.product_url %}
     protocol=https
     > [Press Return]
```

**Review by subatoi — COMMENTED** [review]
Hi @joaoreis13 and thanks for raising this—I agree this needs fixing, but to keep it as simple as possible and for the sake of consistency, I'd prefer to document it the same way as we do here:

https://docs.github.com/en/get-started/getting-started-with-git/updating-credentials-from-the-macos-keychain#deleting-your-credentials-via-the-command-line

I've made a couple of suggestions—if you're happy with those, we can go ahead and get this merged 👍

File: content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts.md [hunk 3]
```diff
@@ -52,10 +52,15 @@ Alternatively, if you want to use the HTTPS protocol for both accounts, you can
...
+   * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, enter the following command and interactive inputs:
 
      ```shell copy
-     git credential-osxkeychain erase https://github.com
+     ```suggestion
+     $ git credential-osxkeychain erase
+     host={% data variables.product.product_url %}
+     protocol=https
+     > [Press Return]
+     >
```
Comment (janbrasna) [inline content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts.md:63]: fyi the suggestion got somehow applied wrong, this needs a little edit:

```suggestion
     $ git credential-osxkeychain erase
     host={% data variables.product.product_url %}
     protocol=https
     > [Press Return]
     >
```

File: content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts.md [hunk 4]
```diff
@@ -52,10 +52,15 @@ Alternatively, if you want to use the HTTPS protocol for both accounts, you can
 {% data reusables.git.confirm-credential-manager %}
 {% data reusables.git.clear-the-stored-credentials %}
    {% data reusables.git.no-credential-manager %}
-   * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, enter the following command.
+   * If the output is `osxkeychain`, you're using the macOS keychain. To clear the credentials, enter the following command and interactive inputs:
 
      ```shell copy
```
Comment (janbrasna) [inline content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts.md:57]: also when mixing prompt and output, the whole example should not be copied perhaps?
```suggestion
     ```shell
```

**Review by nguyenalex836 — DISMISSED** [review]

**Review by nguyenalex836 — APPROVED** [review]

**Pull request merged**